projects/librdkafka.cmake

```diff
@@ -12,14 +12,15 @@ set(PRO_LIBRDKAFKA
   DESC "librdkafka is a C library implementation of the Apache Kafka protocol, containing both Producer and Consumer support -- [windows-only patch](../patches/librdkafka-windows.patch)"
   REPO "repo" ${REPO} "distributePro fork of librdkafka repo on github"
   VER ${VER}
-  GIT_ORIGIN git://github.com/distributepro/librdkafka.git
+  GIT_ORIGIN git://github.com/edenhill/librdkafka.git
   GIT_UPSTREAM git://github.com/edenhill/librdkafka.git
-  GIT_TAG xp-${VER} # what to 'git checkout'
+  GIT_TAG ${VER} # what to 'git checkout'
   GIT_REF ${VER} # create path from this tag to 'git checkout'
   DLURL ${REPO_UPSTREAM}/archive/${VER}.tar.gz
-  DLMD5 8e5685baa01554108ae8c8e9c97dc495
+  https://github.com/edenhill/librdkafka/archive/v1.5.3.tar.gz
```

Comment:
What's this URL about?

Reply:
ah. copy/paste error. Sorry, ill remove it